contents/docs/llm-analytics/generations.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Generations
+availability:
+  free: full
+  selfServe: full
+  enterprise: full
+---
+
+import GenerationEvent from "./_snippets/generation-event.mdx"
+
+Generations are an event that capture an LLM request. The [generations tab](https://app.posthog.com/llm-observability/generations) lists them along with the properties autocaptured by PostHog like the person, model, total cost, token usage, and more.
+
+When you expand a generation, it includes the properties and metadata that every event has along with a conversation history with the role (system, user, assistant) as well as input and output content.
```

Comment:
Include a blurb about how you can use these generations during analysis like other events? I assume you can build insights on these, too.